Create README.md

votuanphong1983 opened the pull request.

votuanphong1983 committed:
Create README.md

pandafox commented on README.md:
```diff
@@ -0,0 +1,71 @@
+` cd ja`
+
+ Use any text editor to open file index.json and start translating...
+ This is json format file, we will translate the value field of this file. E.g:
```
Please indicate that the file should be stored with utf8 Unicode format

pandafox commented on README.md:
```diff
@@ -0,0 +1,71 @@
+
+![](https://content.screencast.com/users/VoTuanPhong/folders/Jing/media/97c32142-8779-43ce-b8ad-cc3ebc1830e6/2018-04-09_1509.png)
+
+Everything is done. We will review your pull request and merge it into our Crypto Miner
```
Indicate that we would receive notification of the pull request, and we'll
1. review and test the code
2. request for changes
3. once all that's done, we'll merge and update our code

votuanphong1983 committed:
Update Readme.md

1. Add 'index.json' should be utf8 unicode
2. add  'We would received notificatino...'

pandafox commented on README.md:
```diff
@@ -35,17 +35,19 @@ Please refer [iso-2-letter-language-codes](https://www.sitepoint.com/iso-2-lette
 
  Use any text editor to open file index.json and start translating...
- This is json format file, we will translate the value field of this file. E.g:
+ This is json utf unicode format file, we will translate the value field of this file. E.g:
```
utf8 to be exact

pandafox commented on README.md:
```diff
@@ -35,17 +35,19 @@ Please refer [iso-2-letter-language-codes](https://www.sitepoint.com/iso-2-lette
  ![](https://content.screencast.com/users/VoTuanPhong/folders/Jing/media/4f72af9d-7d63-44c6-a60c-203ecf9db456/2018-04-09_1440.png)
 
 
+_Note: we have to make sure that index.json is stored in utf8 unicode_
```
utf8 to be exact

votuanphong1983 committed:
change to 'utf8'

change to 'utf8'
